package.json

```diff
@@ -65,7 +65,6 @@
         "mapbox-gl": "^2.4.1",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
-        "react-ga": "^3.3.0",
```

Comment:
Why not use https://github.com/codler/react-ga4

Reply:
@tnagorra this library is a new one. previously it was https://github.com/PriceRunner/react-ga4 which was unmaintained. I will integrate the mentioned library.